content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -71,32 +76,59 @@ Do not use trailing slashes when specifying the path to the packages directory.
 
 ## Build the O3DE project
 
-Use CMake to build the Visual Studio project in the build directory of your O3DE project.
+Use CMake to build the Visual Studio project in the build directory of your O3DE project. For instructions, choose the tab that corresponds to your engine build type. If you installed O3DE, choose "Pre-Built SDK Engine".
+
+{{< tabs name="Project Manager locations" >}}
+{{% tab name="Source Engine" %}}
 
-1. Build the project launcher using the solution you created in the `build/windows_vs2019` directory. The `profile` build configuration is shown in this example.
+1. Build the project launcher and **O3DE Editor** using the solution you created in the `build/windows_vs2019` directory. The `profile` build configuration is shown in this example.
+
+    ```cmd
+    cmake --build build/windows_vs2019 --target MyProject.GameLauncher Editor --config profile -- /m
+    ```
+
+    When specifying `Editor` as a build target, the **Asset Processor** and **Project Manager** will be built too, since they are dependencies of the **Editor**.
+
+    The `/m` is a recommended build tool optimization, which tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
+
+1. When the build is complete, the engine and project binaries can be found under `build/windows_vs2019/bin/profile` in the project directory.
+
+1. Run **O3DE Editor** to verify the project is ready to use.
+
+    ```cmd
+    build\windows_vs2019\bin\profile\Editor.exe
+    ```
+
+    You can also run **Project Manager** (`o3de.exe`) from the same directory to edit your project's settings, add or remove Gems from the project, rebuild your project, and launch the Editor.
+
+{{% /tab %}}
+{{% tab name="Pre-Built SDK Engine" %}}
+
+1. Build the project launcher using the solution you created in the project's `build/windows_vs2019` directory. The `profile` build configuration is shown in this example.
 
     ```cmd
     cmake --build build/windows_vs2019 --target MyProject.GameLauncher --config profile -- /m
     ```
```

Comment:
In the Pre-Built SDK workflow we should also recommend that users build the Editor as well.
Previously there was no Editor target available to build, but we added one to address an important limitation with the pre-built SDK.

Adding gems with Editor target to the project would result in the Editor targets not being built, since the GameLauncher doesn't depend on those targets.

With this we can have the same workflow of building the Project with the SDK Engine as we do the Project Engine

```cmd
    cmake --build build/windows_vs2019 --target MyProject.GameLauncher Editor --config profile -- /m
 ```

Reply:
@lumberyard-employee-dm Does Project Manager do this too, when run from a pre-built SDK engine?

Reply:
Yes it does actually.
We do it in this file: https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Platform/Windows/ProjectBuilderWorker_windows.cpp#L144-L145